universal-application-tool-0.0.1/app/services/question/QuestionType.java

```diff
@@ -23,10 +23,11 @@ public boolean isMultiOptionType() {
   }
 
   public static QuestionType of(String name) throws InvalidQuestionTypeException {
+    String upperName = name.toUpperCase();
```

Comment:
Is there a reason not to just use `name = name.toUpperCase()`?
Since Java is weird it just creates a new reference, so you're not modifying the value.

Reply:
No reason other than it's the style I normally write in. Feels more readable to me to be like "this is a new thing." LMK if you feel strongly though!